### DISCUSSION TITLE
is there any option to open two calender at a time with range date?

### DISCUSSION THREAD

**Discussion by mdnafeed** [primefaces/community#80] [PrimeNG] [answered]
`<p-calendar [(ngModel)]="rangeDates" selectionMode="range" [readonlyInput]="true"></p-calendar>`

Example: 

![image](https://github.com/primefaces/community/assets/17257897/c062af50-bd2a-442b-84b9-91fd4e5353d2)

**Comment by gucal** [comment, marked as answer]
Hi,

You can use the `numberOfMonths` props.

This will work for you.
`<p-calendar [(ngModel)]="rangeDates" selectionMode="range" [readonlyInput]="true" [numberOfMonths]="3"></p-calendar>`

Best regards!

**Comment by mdnafeed** [comment]
Thank you!!

Now working fine
